### PR TITLE
[skip ci] update install from code

### LINKED ISSUE #3810: What is milvus::CollectionParam?
**Describe the bug**
Since `CreateCollection(const Mapping& mapping, const std::string& extra_params) = 0;` doesn't work as I stated [here](https://github.com/milvus-io/milvus/issues/3809) on my device, I wanna switch to `CreateCollection(collection_param);` as shown in the [example](https://github.com/milvus-io/milvus/blob/836a247b6ab9cd023c16b5c0a2c799adb88157f1/sdk/examples/qps/src/ClientTest.cpp#L146) . However, I didn't find the definition of either the function or the struct. 

However, I saw some people said they used it before, but there was no code for reference. So may I ask what it is and where I can find it? How should I use it? (I think the method should be similar to  `CreateCollection(const Mapping& mapping, const std::string& extra_params) = 0;`, but since it didn't work on my device, I have no idea).

Thanks for any possible help!

### PR DESCRIPTION
**Pull request by shengjun1985**
Signed-off-by: shengjun.li <shengjun.li@zilliz.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Fixes #3810 

**What this PR does / why we need it:**
(1) add "Specify version"
(2) add CentOs7 docker image
(3) modify "Build Milvus source code"

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Review by wxyucs — CHANGES_REQUESTED** [review]
Why not specified the label of milvus build env docker image?

File: INSTALL.md [hunk 1]
```diff
@@ -130,26 +158,41 @@ Pull GPU-enabled image:
...
 ```shell
-$ docker run --runtime=nvidia -it -p 19530:19530 -d milvusdb/milvus-gpu-build-env:latest
+$ docker run --runtime=nvidia -it -p 19530:19530 -d [milvus_gpu_docker_image]
 ```
 
 - For nvidia container toolkit:
 
 ```shell
-docker run --gpus all -it -p 19530:19530 -d milvusdb/milvus-gpu-build-env:latest
+docker run --gpus all -it -p 19530:19530 -d [milvus_gpu_docker_image]
```
[the remark this answers is not in the document]
Comment (shengjun1985) [reply INSTALL.md]: done

File: INSTALL.md [hunk 2]
```diff
@@ -130,26 +158,41 @@ Pull GPU-enabled image:
...
+$ docker run -it -p 19530:19530 -d [milvus_cpu_docker_image]
 ```
 
 Start a GPU container:
 
 - For nvidia docker 2:
 
 ```shell
-$ docker run --runtime=nvidia -it -p 19530:19530 -d milvusdb/milvus-gpu-build-env:latest
+$ docker run --runtime=nvidia -it -p 19530:19530 -d [milvus_gpu_docker_image]
```
[the remark this answers is not in the document]
Comment (shengjun1985) [reply INSTALL.md]: done

File: INSTALL.md [hunk 3]
```diff
@@ -130,26 +158,41 @@ Pull GPU-enabled image:
...
+$ docker pull milvusdb/milvus-gpu-build-env:v0.7.0-centos7
+```
+
 ### Step 2 Start the Docker container
 
 Start a CPU-only container:
 
 ```shell
-$ docker run -it -p 19530:19530 -d milvusdb/milvus-cpu-build-env:latest
+$ docker run -it -p 19530:19530 -d [milvus_cpu_docker_image]
```
[the remark this answers is not in the document]
Comment (shengjun1985) [reply INSTALL.md]: done

File: INSTALL.md [hunk 4]
```diff
@@ -119,6 +145,8 @@ With the following Docker images, you should be able to compile Milvus on any Li
 
 ### Step 1 Pull Milvus Docker images
 
+### ubuntu18.04
```
[the remark this answers is not in the document]
Comment (shengjun1985) [reply INSTALL.md]: done

**Review by wxyucs — APPROVED** [review]

**Pull request merged**